install_bootstrap

```diff
@@ -260,6 +263,15 @@ while [[ "${run_dockstore_launcher^^}" != 'Y' &&  "${run_dockstore_launcher^^}"
         ui2_hash='ui2_hash'
       	ask_question "What Dockstore UI 2 CDN hash do you want to use (ex: 2.4.0-beta.3-70bb93a )?" "$UI2_HASH" "Dockstore hash" $ui2_hash
 
+        database_domain='database_domain'
```

Comment:
So I've been modifying the generated config file instead of answering these questions so I can press enter all the way down since the very beginning.  Does anyone here actually answer these questions anymore?  it's quite lengthy nowdays

Reply:
I just answer the ones that changed

Reply:
dockstore-deploy is essentially doing the same thing by generating the config file, and the runs `install_bootstrap --script`, so the questions don't even appear.

But someday, who knows, maybe there will be a need.